src/Autumn/compile.jl

```diff
@@ -4,11 +4,140 @@ module Compile
 using ..AExpressions
 using ..Program
 
+export compileToJulia, toRepr
+
+struct AutumnCompileError <: Exception
+  msg
+end
+
+fixedSymbols = [:+, :-, :/, :*, :&&, :||, :>=, :<=, :>, :<, :(==)]
+
+AutumnCompileError() = AutumnCompileError("")
+
 "Compile `aexpr` into `program::Program`"
-function compiletojulia(::AExpr)::AProgram
-  # Do type inference
-  # Get external values
-  # Alot
+function compileToJulia(aexpr::AExpr) #::AProgram
+
+  ### DATA ###
+  data = Dict([("historyVars" => []),
+               ("externalVars" => []),
+               ("initnextVars" => []),
+               ("initnextOther" => []),
+               ("types" => [])])
+
+  ### HELPER FUNCTION ###
+  function toRepr(expr::AExpr, parent=Nothing)::String
```

Comment:
The big issue is that this compiler returns a `String`.  It should return an `Expr`.
Julia can eval `Expr` directly.

Reply:
The helper function `toRepr` returns a string, but `Meta.parse` is called on the string to convert it to an expr when the outer function `compiletojulia` returns.

Reply:
I see.  Still I think this is an odd way to go about things for a few reasons:

1. It makes your code more verbose
2. You have to spend the extra cost generating a string then parsing it, which may or may not be negligible when we are executing this repeatedly.
4. You'll have to put in extra work to handle things like parentheses, which are not necessary in the abstract syntax tree
5. Most important: it makes me feel a little icky -- we shouldn't be working with strings when we don't need to